dmutils/forms.py

```diff
@@ -98,7 +98,7 @@ def csrf_context(self):
 
     def __init__(self, *args, **kwargs):
         if current_app.config['CSRF_ENABLED']:
-            self.Meta.csrf_secret = current_app.config['SECRET_KEY']
+            self.Meta.csrf_secret = current_app.config['SECRET_KEY'].encode()
```

Comment:
not sure if we need encode() here

Reply:
The csrf_secret needs to be a [bytes string](https://wtforms.readthedocs.io/en/2.3.x/csrf/#wtforms.csrf.session.SessionCSRF).